src/OpenSpaceToolkit/Astrodynamics/EventCondition.cpp

```diff
@@ -10,8 +10,54 @@ namespace ostk
 namespace astro
 {
 
-EventCondition::EventCondition(const String& aName)
-    : name_(aName)
+EventCondition::Target::Target(const Real& aValue, const Type& aType)
+    : value_(aValue),
+      type_(aType)
+{
+}
+
+EventCondition::Target::Target(const Angle& anAngle, const Type& aType)
```

Comment:
Man this is a kind-of annoying downside to using the OSTk native units

Reply:
Yeah it's unfortunate, but atleast it's overloaded so there's that...